RELEASE-NOTES.md

```diff
@@ -160,6 +160,22 @@ In this way it will be possible to connect external RNF models to the ruby inter
 * Add support for Zicbop extension #1710
 * Fix vector instruction assertion caused by speculative execution #1711
 
+## GPU model improvements
+
+The GPU model is now available in the standard library!
+There is a new `ViperBoard` in `gem5.prebuilt.viper`.
+This board is an initial implementation and will be improved in the next versions of gem5.
+There is an example script in `configs/example/gem5_library/x86-mi300x-gpu.py` that shows how to use the `ViperBoard`.
+See #1636.
+
+### Other GPU changes
+
+* Vega10 has been deprecated #1619
+* Replacement policy has been improved #1564
+* Improvements to the TLB #1431
```

Comment:
I think @mattsinc is planning to revert this with #1769 .

Reply:
I remove this line based on this comment. Probably best not advertise things that will get reverted.

Reply:
Thanks @abmerop , I just merged #1769 in after manually making the requested changes.